### ISSUE TITLE
v3-set-env command unable to set value begins with `-`

### ISSUE THREAD

**Issue by nusnewob** [cloudfoundry/cli#1277]
`v3-set-env` command unable to set env var value begins with `-`

## Command  

`cf v3-set-env java_app JAVA_OPTS -Dconfig`
`cf v3-set-env java_app JAVA_OPTS \-Dconfig`
`cf v3-set-env java_app JAVA_OPTS "-Dconfig"`

## What occurred

```
Incorrect Usage: unknown flag `D'
```

`cf v3-set-env java_app JAVA_OPTS "\-Dconfig"` works, but the CLI also adds the escape char to the env var,
```
$ cf v3-set-env java_app
...
User-Provided:
JAVA_OPTS: "\\-Dconfig"
```

## CLI Version  

`cf version 6.33.0+a345ea34d.2017-11-20`

## CC API Endpoint Version  

```
api endpoint:   https://api_endpoint
api version:    2.99.0
```

## CF Trace

no stack trace available

## Platform & Shell Details  

Mac OS X 10.13.1 
iTerm Build 3.1.5.beta.2

**Comment by dkoper** [comment]
Thanks! We've fixed the issue, will be in the next release.

**Comment by dkoper** [comment]
Fixed in cf CLI 6.33.1. Thanks again for the report.
